lerna.json

```diff
@@ -1,6 +1,16 @@
 {
+  "version": "independent",
+  "npmClient": "npm",
   "packages": [
-    "packages/*"
+    "src/components/*"
   ],
-  "version": "0.0.0"
+  "command": {
+    "publish": {
+      "ignoreChanges": [
+        "*.stories.ts",
+        "*.md"
+      ],
+      "message": "chore(release): publish"
```

Comment:
We might want to change this. Anyone have an opinion?

Reply:
Something like "Component version updated by Lerna" ?

Reply:
Version will not be updated here since I changed versioning independently for each component `"version": "independent"`. It will be in the package.json of each component